spec/hook_context_extension_spec.rb

```diff
@@ -120,23 +121,23 @@
       end
 
       after do
-        expect(Kafo::PuppetCommand).to have_received(:search_puppet_path).twice
+        expect(Kafo::PuppetCommand).to have_received(:search_puppet_path).twice # rubocop:disable RSpec/ExpectInHook
```

Comment:
This won't be needed until we update RuboCop
```suggestion
        expect(Kafo::PuppetCommand).to have_received(:search_puppet_path).twice
```